example/cumulus-tf/variables.tf

```diff
@@ -286,3 +286,9 @@ variable "rds_connection_heartbeat" {
   type        = bool
   default     = false
 }
+
+variable "ems_deploy" {
+  description = "If true, deploys the EMS reporting module"
+  type        = bool
+  default     = true
```

Comment:
So it should be default false but this is just for the example deployment?

Reply:
Correct. This is just so the tests pass for now.